r/src/Makevars.in

```diff
@@ -16,6 +16,6 @@
 # under the License.
 
 PKG_CPPFLAGS=@cflags@
-PKG_CXXFLAGS=$(C_VISIBILITY)
+PKG_CXXFLAGS+=$(C_VISIBILITY)
```

Comment:
This resolves the ABI incompat, but alsoe results:

```
* checking for GNU extensions in Makefiles ... WARNING
Found the following file(s) containing GNU extensions:
  src/Makevars
  src/Makevars.in
Portable Makefiles do not use GNU extensions such as +=, :=, $(shell),
$(wildcard), ifeq ... endif. See section 'Writing portable packages' in
the 'Writing R Extensions' manual.
```

Reply:
Is this portability warning at all related to why appveyor tests on this PR are now failing?